src/nodes/QskBoxRendererEllipse.cpp

```diff
@@ -641,6 +682,91 @@ namespace
                             c[ corner ].centerY + v.dy2( corner ),
                             borderMapBR.colorAt( j ) );
                     }
+
+                    // at the beginning and end of the loop we can add
+                    // additional lines for border gradients:
+
+                    if( j == 0 )
+                    {
+                        if( additionalGradientStops( borderMapTR.gradient() ) > 0 )
+                        {
+                            auto stops = borderMapTR.gradient().stops();
```

Comment:
const auto stops

Reply:
It turns out we didn't use the variable at all, so I removed them.